ruleset.xml

```diff
@@ -1,93 +1,468 @@
+  -->
+  <!-- TODO: Create PR to fix this rule -->
+  <!--
+  <rule ref="category/java/bestpractices.xml/AvoidMessageDigestField"/>
+  -->
+  <rule ref="category/java/bestpractices.xml/AvoidPrintStackTrace"/>
+  <rule ref="category/java/bestpractices.xml/AvoidReassigningCatchVariables"/>
+  <!-- TODO: Create PR to fix this rule -->
+  <!--
+  <rule ref="category/java/bestpractices.xml/AvoidReassigningLoopVariables">
+    <properties>
+      <property name="foreachReassign" value="allow"/>
+    </properties>
+  </rule>
+  -->
+  <rule ref="category/java/bestpractices.xml/CheckResultSet"/>
+  <rule ref="category/java/bestpractices.xml/ConstantsInInterface"/>
+  <rule ref="category/java/bestpractices.xml/DefaultLabelNotLastInSwitch"/>
+  <rule ref="category/java/bestpractices.xml/DoubleBraceInitialization"/>
+  <rule ref="category/java/bestpractices.xml/ForLoopCanBeForeach"/>
+  <!-- TODO: Create PR to fix this rule -->
+  <!--
+  <rule ref="category/java/bestpractices.xml/ForLoopVariableCount"/>
+  -->
+  <!-- TODO: Create PR to fix this rule -->
+  <!--
+  <rule ref="category/java/bestpractices.xml/GuardLogStatement"/>
+  -->
+  <rule ref="category/java/bestpractices.xml/JUnit4SuitesShouldUseSuiteAnnotation"/>
+  <rule ref="category/java/bestpractices.xml/UnitTestShouldUseAfterAnnotation"/>
+  <rule ref="category/java/bestpractices.xml/UnitTestShouldUseBeforeAnnotation"/>
+  <!-- TODO: Create PR to fix this rule -->
+  <!--
+  <rule ref="category/java/bestpractices.xml/UnitTestShouldUseTestAnnotation"/>
+  -->
+  <rule ref="category/java/bestpractices.xml/JUnit5TestShouldBePackagePrivate"/>
+  <rule ref="category/java/bestpractices.xml/UnitTestShouldIncludeAssert"/>
+  <!-- TODO: Create PR to fix this rule -->
+  <!--
+  <rule ref="category/java/bestpractices.xml/LooseCoupling"/>
+  -->
+  <!-- TODO: Create PR to fix this rule -->
+  <!--
+  <rule ref="category/java/bestpractices.xml/MethodReturnsInternalArray"/>
+  -->
+  <!-- TODO: Create PR to fix this rule -->
+  <!--
+  <rule ref="category/java/bestpractices.xml/MissingOverride"/>
+  -->
+  <rule ref="category/java/bestpractices.xml/OneDeclarationPerLine"/>
+  <!-- TODO: Create PR to fix this rule -->
+  <!--
+  <rule ref="category/java/bestpractices.xml/PreserveStackTrace"/>
+  -->
+  <rule ref="category/java/bestpractices.xml/PrimitiveWrapperInstantiation"/>
+  <rule ref="category/java/bestpractices.xml/ReplaceEnumerationWithIterator"/>
+  <!-- TODO: Create PR to fix this rule -->
+  <!--
+  <rule ref="category/java/bestpractices.xml/ReplaceHashtableWithMap"/>
+  -->
+  <!-- TODO: Create PR to fix this rule -->
+  <!--
+  <rule ref="category/java/bestpractices.xml/ReplaceVectorWithList"/>
+  -->
+  <rule ref="category/java/bestpractices.xml/SimplifiableTestAssertion"/>
+  <rule ref="category/java/bestpractices.xml/NonExhaustiveSwitch"/>
+  <!-- TODO: Create PR to fix this rule -->
+  <!--
+  <rule ref="category/java/bestpractices.xml/SystemPrintln" />
```

Comment:
We had defined exceptions, perhaps they should be defined directly here or a general comment.

Reply:
This is up to the PR to enable the rule and implement exceptions in the class